docs/guide/connection-settings.txt

```diff
@@ -1,3 +1,286 @@
 =======================
 URI Connection Settings
 =======================
+
+Optional connection settings are settings not covered by the :manual:`URI Connection String </reference/connection-string/>`. The following options are passed in the options parameter when you create a mongo client.
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   const assert = require('assert');
+
+   // Connection URL
+   const url = 'mongodb://localhost:50000,localhost:50001';
+   // Database Name
+   const dbName = 'myproject';
+
+   // create a client, passing in additional options
+   const client = new MongoClient(url, {
+     poolSize: 10, ssl: true
+   });
+
+   // Use connect method to connect to the server
+   client.connect.then(async function() {
+     console.log("Connected correctly to server");
```

Comment:
Not required for LGTM, but nice to have:

Could we either use singlequotes or doublequotes on these code-snippets? 

Note - I've definitely been guilty of using a mix of single & double quotes in the past, so this is a little hypocritical for me to comment haha. Running the snippets through eslint would be a great help